src/mlmc/simulation.py

```diff
@@ -1,11 +1,11 @@
 import numpy as np
+import os, glob, shutil
 
 
 class Simulation:
     """
     Parent class for simulations
```

Comment:
Describe that this is a base class which should be inherited. Which methods has to be implemented by child class?